### PR TITLE
add llama3 8b config

### PR DESCRIPTION
**Pull request by lchu-ibm**
Add llama3 8b config.

we also expose vocab_size to make our dummy dataloader configurable with llama3 so we can overwrite default 32k to 128k.

### CODE REVIEW

File: fms_fsdp/utils/config_utils.py [hunk 1]
```diff
@@ -55,6 +55,26 @@ def get_model_config(model_variant):
...
+    elif model_variant == "8b":
+        llama_config = LLaMAConfig(
+            src_vocab_size=128256,
+            emb_dim=4096,
+            nheads=32,
+            kvheads=8,
+            nlayers=32,
+            hidden_grow_factor=3.5,
+            max_expected_seq_len=8192,
+        )
```
Comment (daviswer) [inline fms_fsdp/utils/config_utils.py:67]: Do we not have support for these model configs already in main FMS? Should we be adding these here?

Comment (lchu-ibm) [reply fms_fsdp/utils/config_utils.py:67]: `get_model` is slightly complex to this use case.   the remote FMS repo was also not the latest.  so for configuration I prefer doing it separately here.

Comment (daviswer) [reply fms_fsdp/utils/config_utils.py:67]: Oh, fair enough. We should probably reconcile these at some point though

File: fms_fsdp/utils/dataloader_utils.py [hunk 2]
```diff
@@ -20,8 +20,6 @@ class SteadyCounter(torch.utils.data.IterableDataset):
         # Spit out incremental counts of constant length l, modulo vocab size v
         def __init__(self, l, v):
             self.i = 0
-            self.rank = 0
-            self.worldsize = 1
```
Comment (daviswer) [inline fms_fsdp/utils/dataloader_utils.py:24]: Noting that we will need `rank` and `worldsize` if we ever want to wrap `SteadyCounter` in any of the existing non-dummy dataset wrappers

**Review by daviswer — APPROVED** [review]

**Pull request merged**